pandas/tests/io/formats/test_format.py

```diff
@@ -2846,6 +2846,13 @@ def test_to_string_multindex_header(self):
         exp = "    r1 r2\na b      \n0 1  2  3"
         assert res == exp
 
+    def test_to_string_empty_col(self):
```

Comment:
could parametrize:
``` python
pytest.mark.parametrize(
    "data, expected",
    (
        ["", "Hello", "World", "", "", "Mooooo", "", ""],
        "      \n Hello\n World\n      \n      \nMooooo\n      \n      ",
    ),
)
def test...

```

Reply:
Second argument would have to be a list of tuples rather than a single tuple in order to work. I think since there's only one case there's no reason to parametrize?

Reply:
yes that's right, would need add a comma there to make the arg a tuple 

but also yeah i'm fine with not parametrizing too since there's only one argument. personally i find this slightly easier to look at but I don't mind either way

Reply:
reverted this to the earlier un-parametrized version